crates/storage/src/kv_store.rs

```diff
@@ -62,6 +66,15 @@ pub trait KeyValueInspect {
     /// Returns the value from the storage.
     fn get(&self, key: &[u8], column: Self::Column) -> StorageResult<Option<Value>>;
 
+    /// Returns multiple values from the storage.
+    fn get_batch<'a>(
+        &'a self,
+        keys: BoxedIter<'a, Vec<u8>>,
```

Comment:
It is very sad that we need to convert `&[u8]` into a vector. Are you use that we can't pass `Cow<'a, [u8]>`?

Reply:
Thanks for suggesting. Yeah I think a `Cow` can be used here. Originally I wanted to take `impl AsRef<[u8]>` but again that's a problem with object safety.

I'll see if I hit any walls with the `Cow` solution.